### DISCUSSION TITLE
Help Getting Started with CE C Toolchain

### DISCUSSION THREAD

**Discussion by Priyanshu4** [CE-Programming/toolchain#339] [Q&A] [answered]
Hi, I just installed the CE C Toolchain. I'm following the instructions from the following link https://ce-programming.github.io/toolchain/v9.2/static/getting-started.html, but when I run make in the hello world directory, I get the error that CEdev\bin\fasmg.exe is not recognized. I'm using a windows 10 pc. Does anyone know how to fix this? Thanks in advance.

**Comment by Priyanshu4** [comment, marked as answer]
It turns out my antivirus was just deleting the file. I no longer need help. Thanks.

**Comment by adriweb** [comment]
fasmg is indeed known for being a false positive with antivirus software :/
